image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -212,8 +220,11 @@
 {{/* Image settings */}}
 {{ include "srox.configureImage" (list $ ._rox.image.main) }}
 {{ include "srox.configureImage" (list $ ._rox.image.collector) }}
-[< if (not .KubectlOutput) ->]
+[<- if not .KubectlOutput >]
 {{ include "srox.configureImage" (list $ ._rox.image.scanner) }}
+[<- if and .FeatureFlags.ROX_SCANNER_V4 (not .KubectlOutput) >]
```

Comment:
I think `(not .KubectlOutput)` is already true?

Reply:
removed